gen_dungeons.lua

```diff
@@ -858,13 +858,13 @@ rewards[36] = {
 }
 rewards[37] = {
   [0]={[0]={ore=2, cards={[210056]=1}}},
-  [10]={[0]={cards={[210056]=45,[210050]=12,[210051]=12,[210052]=12}},
+  [10]={[0]={cards={[210056]=50,[210051]=12,[210052]=12,[210053]=12}},
```

Comment:
I didn't document anything because I never document things, but the way this works is there are 100 actual slots and the "0" slot. The slots that are not filled in get filled with the value in the previous slot. Slots can specify an index to copy the value from that index. Clears after 100 use the "0" slot.

So you'll need the `[21]=0,` to get it to start using the "0" setting at the 21st clear.

Reply:
Like in the newest commit (a1f2f54)?

Reply:
yeah!